### PR TITLE
This is fixes for the GPU ARM template.

### PR DESCRIPTION
**Pull request by chmitch**
The Nvidia drivers weren't installed on the prior checkin.  This update switches the template VM to the DSVM image which is preconfigured for all the different classes of GPU VMs.

### CODE REVIEW

File: deploy/azure/azuredeploy_gpu.json [hunk 1]
```diff
@@ -241,7 +232,7 @@
           "typeHandlerVersion": "2.1",
           "autoUpgradeMinorVersion": true,
           "settings": {
-            "fileUris": ["https://raw.githubusercontent.com/likamrat/foldingathome/master/deploy/azure/install_fah_gpu.sh"],  
+            "fileUris": ["https://raw.githubusercontent.com/chmitch/foldingathome/master/deploy/azure/install_fah_gpu.sh"],
```
Comment (likamrat) [inline deploy/azure/azuredeploy_gpu.json:237]: Make sure it's pointing to the right repo

**Comment by chmitch** [comment]
Updating this PR.   Fixed the errant file path to point to the right repo.   I also fixed the shell script to install nvidia drivers and reverted back to the base ubuntu image.

File: docs/AzureGPU.md [hunk 2]
```diff
@@ -20,10 +20,18 @@ Example: ```az group create -l eastus -n GPU-Folding-COVID19```
...
 * Now that you have a new RG created, you can go ahead and deploy the GPU VM in a single command: <br>
 
-```az deployment group create --name "folding" --resource-group "GPU-Folding-COVID19" --template-uri https://raw.githubusercontent.com/likamrat/foldingathome/master/deploy/azure/azuredeploy_gpu.json```
+```az deployment group create --name "folding" --resource-group "GPU-Folding-COVID19" --template-uri https://raw.githubusercontent.com/likamrat/foldingathome/master/deploy/azure/azuredeploy_gpu.json --parameters https://raw.githubusercontent.com/likamrat/foldingathome/master/deploy/azure/azuredeploy_gpu.parameters.json```
 
 **Note: Don't forget to match the resource group name based on the one you created**
 
+# Advanced VM Deployment
+
+The ARM Template leverages a parameters file with preconfigured values for things like username and password.   While you can clone this repo and make the necessary modifications to the files to override the defaults, there is a quicker path leveraging the same ```--parameters``` cli argument.  For example, the following command would do the same deployment with a customized username and password:
```
Comment (likamrat) [inline docs/AzureGPU.md:29]: Extra space after the first sentence

File: docs/AzureGPU.md [hunk 3]
```diff
@@ -20,10 +20,18 @@ Example: ```az group create -l eastus -n GPU-Folding-COVID19```
...
+
+```az deployment group create --name "folding" --resource-group "Folding-COVID19" --template-uri https://raw.githubusercontent.com/likamrat/foldingathome/master/deploy/azure/azuredeploy_gpu.json --parameters adminUsername="<some username>" adminPasswordOrKey="<some strong password>" ```
+
+**Note:  You'll need to replace ```<some username>``` and ```<some strong password>``` with your own values.**
+
 ![](../img/Azure/az_01.png)
 
 ![](../img/Azure/az_02.png)
```
Comment (likamrat) [inline docs/AzureGPU.md:37]: Fix the "Reboot & Shutdown" section at the end of the doc. Currently, it is aligned with Windows.

**Review by likamrat — CHANGES_REQUESTED** [review]
Fix "Reboot & Shutdown" section

**Review by likamrat — APPROVED** [review]
LGTM

**Pull request merged**